cityofaustin/atd-moped
# Ability to map a signal component instead of picking from list

This PR depends on https://github.com/cityofaustin/atd-moped/pull/1094.

## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/12998

## Testing
**URL to test:**  Local

1. Create a signal by picking it from the signal autocomplete input (should work normally)
2. Edit the signal component by clearing the selected signal from the autocomplete input (should result in unmapped component)
3. Map the unmapped signal component (location should be saved as expected)
4. Edit the the signal component you just mapped by picking a signal from the list (the location should be updated with the location you selected)
5. Edit the signal component again by changing it to an **Intersection - Improvement** component (this will result in the component being unmapped)
6. Edit the **Intersection - Improvement** component by switching to a signal component, but leave the signal input blank (should result in the location being unchanged).


---
#### Ship list
